### PR TITLE
allow user to specify scan column

### PR DESCRIPTION
**Pull request by ming535**
Support user defined scan column.

### CODE REVIEW

File: pkg/inputs/mysqlbatch/input.go [hunk 1]
```diff
@@ -26,7 +26,9 @@ type TableConfig struct {
 	Schema string `mapstructure:"schema" toml:"schema" json:"schema"`
 	// Table is an array of string, each string is a glob expression
 	// that describes the table name
-	Table []string `mapstructure:"table" toml:"table" json:"table"`
+	Table string `mapstructure:"table" toml:"table" json:"table"`
```
Comment (Ryan-Git) [inline pkg/inputs/mysqlbatch/input.go]: why can't `ScanColumn` apply to multiple tables?

File: pkg/inputs/mysqlbatch/input.go [hunk 2]
```diff
@@ -406,33 +418,33 @@ func InitTablePosition(db *sql.DB, positionCache position_store.PositionCacheInt
 // Pick primary key, if there is only one primary key
 // If pk not found try using unique index
 // fail
-func DetectScanColumn(sourceDB *sql.DB, dbName string, tableName string, maxFullDumpRowsCount int64) (string, int64, error) {
+func DetectScanColumn(sourceDB *sql.DB, dbName string, tableName string, maxFullDumpRowsCount int64) (string, error) {
 	rowsCount, err := utils.EstimateRowsCount(sourceDB, dbName, tableName)
```
Comment (Ryan-Git) [inline pkg/inputs/mysqlbatch/input.go]: counting row could be moved down

**Review by Ryan-Git — CHANGES_REQUESTED** [review]

File: pkg/inputs/mysqlbatch/input.go [hunk 3]
```diff
@@ -406,33 +419,28 @@ func InitTablePosition(db *sql.DB, positionCache position_store.PositionCacheInt
...
 	}
 
 	if len(uniqueIndexes) > 0 {
-		return uniqueIndexes[0], rowsCount, nil
+		return uniqueIndexes[0], nil
 	}
 
-	if rowsCount < maxFullDumpRowsCount {
-		return "*", rowsCount, nil
+	if estimatedRowsCount < maxFullDumpRowsCount {
```
Comment (Ryan-Git) [inline pkg/inputs/mysqlbatch/input.go:443]: should this be the first condition?

Comment (ming535) [reply pkg/inputs/mysqlbatch/input.go:443]: no, added a comment in the code @Ryan-Git

**Review by Ryan-Git — APPROVED** [review]

**Pull request merged**